### DISCUSSION TITLE
Can't choose OS version (from Big Sur?)

### DISCUSSION THREAD

**Discussion by woodmanb** [grahampugh/erase-install#56] [Q&A]
Hi,

First time posting at GiHub! Hope you all are safe and are having a nice 2021.

I've successfully tested erase-install in the past, today I was going to write a graphical script to be user-friendly, and realized I cannot choose any version prior to Big Sur although they are available.

Current installed version is: 11.1
Aiming to reinstall version: 10.15 (tried 10.14 as well, none of them worked)

`sudo bash erase-install.sh --erase --os=10.15

   [erase-install] v0.17.2 script execution started: jueves, 14 de enero de 2021, 18:07:01 CET
   [erase-install] Caffeinating this script (pid=1934)
   [erase-install] Looking for existing installer
   [find_existing_installer] No valid installer found.
   [erase-install] Starting download process
   [get_installinstallmacos] installinstallmacos.py is in /Library/Management/erase-install
   [run_installinstallmacos] Checking that selected OS 10.15 is available

   [run_installinstallmacos] This command is now being run:

   installinstallmacos.py --workdir=/Library/Management/erase-install --ignore-cache --raw --os=10.15 --validate 

installinstallmacos.py - get macOS installers from the Apple software catalog

This Mac:
Model Identifier : MacBookPro16,1

Bridge ID        : J152fAP
Board ID         : Mac-XXXXXXX (erased for privacy)
OS Version       : 11.1
Build ID         : 20C69

 #  ProductID       Version    Build    Post Date   Title                          
 1  001-15219       10.15.5    19F2200  2020-06-15  macOS Catalina                 
 2  001-68446       10.15.7    19H15    2020-11-11  macOS Catalina                 
 3  001-04366       10.15.4    19E2269  2020-05-04  macOS Catalina                 
 4  061-86291       10.15.3    19D2064  2020-03-23  macOS Catalina                 
 5  041-91758       10.13.6    17G66    2019-10-19  macOS High Sierra              
 6  001-57224       10.15.7    19H4     2020-10-27  macOS Catalina                 
 7  061-26589       10.14.6    18G103   2019-10-14  macOS Mojave                   
 8  001-51042       10.15.7    19H2     2020-09-24  macOS Catalina                 
 9  001-36735       10.15.6    19G2006  2020-08-06  macOS Catalina                 
10  001-86606       11.1       20C69    2020-12-14  macOS Big Sur                  
11  041-88800       10.14.4    18E2034  2019-10-23  macOS Mojave                   
12  041-90855       10.13.5    17F66a   2019-10-23  Install macOS High Sierra Beta 
13  061-26578       10.14.5    18F2059  2019-10-14  macOS Mojave                   
14  001-36801       10.15.6    19G2021  2020-08-12  macOS Catalina                 

OS 10.15 is not available. Run again without --os argument to select a valid build to download.

   [run_installinstallmacos] No disk image found. I guess nothing got downloaded.

`

It says OS 10.15 is not available although its in the list. I've tried pointing specific versions and builds. No luck at all.

Is it because I'm running 11.1? If so, is there any way of doing it using erase-install to avoid all the manual work?

Thanks a lot!

**Comment by adamzeitoun** [comment]
Bump! looking to do the same thing. Need to downgrade from Big Sur to Catalina. Help would be greatly appreciated!! Thanks

**Reply by grahampugh** [reply]
You cannot downgrade using startosinstall. You can download the installer and create an external installation drive with createosinstallmedia, then boot from that drive on the device you wish to downgrade.
